data/eos-autoupdater.conf

```diff
@@ -10,4 +10,5 @@
 #  3 - apply updates
 LastAutomaticStep=3
 IntervalDays=14
+RandomizedDelayDays=7
```

Comment:
Mid-air collision. I missed @ramcq’s comment.

@ramcq: @cosimoc was concerned that normal-internet-connected computers would no longer immediately get updates after being installed.